setup.py

```diff
@@ -1,14 +1,47 @@
+from setuptools import find_packages
 from setuptools import setup
+import os
+import re
 
-setup(name='exponent_server_sdk',
-    version='0.0.1',
+
+HERE = os.path.abspath(os.path.dirname(__file__))
+
+
+README_PATH = os.path.join(HERE, 'README.md')
+try:
+    with open(README_PATH) as fd:
+        README = fd.read()
+except IOError:
+    README = ''
+
+
+def get_version(package_name):
```

Comment:
@ccheever I looked around and the regex method was a popular way to do it. you motivated me to look again, and there is indeed a better way.

```
version=__import__('exponent_server_sdk').__version__,
```